### PR TITLE
DOC: Regenerate examples

### PR DESCRIPTION
**Pull request by bocklund**
* Regenerate changes from examples
* Add checklist PR template (below)

Checklist
* [x] The documentation examples have been regenerated if the Jupyter notebooks in the `examples/` have changed. To regenerate the documentation examples, run `jupyter nbconvert --to rst --output-dir=docs/examples examples/*.ipynb` from the top level directory)

### CODE REVIEW

File: docs/examples/CementiteAnalysis.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ http://dx.doi.org/10.1016/j.calphad.2010.01.004.
 (http://www.sciencedirect.com/science/article/pii/S0364591610000052)
 
 The TDB file used here differs slightly from the published TDB to ensure
-compatibility with pycalphad's TDB parser. All phases except cementite
```
Comment (richardotis) [inline docs/examples/CementiteAnalysis.rst]: This change with commas will break formatting?

File: docs/examples/UsingCalculationResults.rst [hunk 2]
```diff
@@ -39,15 +39,15 @@ dimensions/coordinates:
...
 -  ``T``: temperatures (in K).
 -  ``component``: the string names of the components in the system
--  ``internal_dof``: The internal\_dof (internal degrees of freedom) is
-   the index of the site in any phase's site fraction array. Below the
-   FCC\_A1 phase has the sublattice model (AL, ZN) and thus the
-   internal\_dof are integers 0 and 1 referring to the AL site (index 0)
+-  ``internal_dof``: The internal_dof (internal degrees of freedom) is
+   the index of the site in any phase’s site fraction array. Below the
+   FCC_A1 phase has the sublattice model (AL, ZN) and thus the
+   internal_dof are integers 0 and 1 referring to the AL site (index 0)
```
Comment (richardotis) [inline docs/examples/UsingCalculationResults.rst]: Don't variable names with underscores still need to be escaped?

Comment (bocklund) [reply docs/examples/UsingCalculationResults.rst]: I built the docs locally and everything looks right
Screen Shot 2020-02-06 at 8 29 20 PM

File: docs/examples/CementiteAnalysis.rst (hunk 1 above)
Comment (bocklund) [reply docs/examples/CementiteAnalysis.rst]: I think it's just a pandoc/unicode thing. Again, looks like it produces the same character to me when I generated the docs myself

**Pull request merged**